api/scripts/wazuh-apid.py

```diff
@@ -75,26 +75,37 @@ def start(foreground, root, config_file):
                 logger.info(f"Generated certificate file in WAZUH_PATH/{to_relative_path(api_conf['https']['cert'])}")
 
             # Load SSL context
-            allowed_ssl_ciphers = {
+            allowed_ssl_protocols = {
                 'tls': ssl.PROTOCOL_TLS,
                 'tlsv1': ssl.PROTOCOL_TLSv1,
                 'tlsv1.1': ssl.PROTOCOL_TLSv1_1,
                 'tlsv1.2': ssl.PROTOCOL_TLSv1_2
             }
             try:
-                ssl_cipher = allowed_ssl_ciphers[api_conf['https']['ssl_cipher'].lower()]
+                ssl_protocol = allowed_ssl_protocols[api_conf['https']['ssl_protocol'].lower()]
             except (KeyError, AttributeError):
                 # KeyError: invalid string value
                 # AttributeError: invalid boolean value
-                logger.error(str(APIError(2003, details='SSL cipher is not valid. Allowed values: '
+                logger.error(str(APIError(2003, details='SSL protocol is not valid. Allowed values: '
                                                         'TLS, TLSv1, TLSv1.1, TLSv1.2')))
                 sys.exit(1)
-            ssl_context = ssl.SSLContext(protocol=ssl_cipher)
+
+            ssl_context = ssl.SSLContext(protocol=ssl_protocol)
             if api_conf['https']['use_ca']:
                 ssl_context.verify_mode = ssl.CERT_REQUIRED
                 ssl_context.load_verify_locations(api_conf['https']['ca'])
             ssl_context.load_cert_chain(certfile=api_conf['https']['cert'],
                                         keyfile=api_conf['https']['key'])
+
+            # Base configuration should not have ciphers?
+            if 'ssl_ciphers' in api_conf['https']:
```

Comment:
ssl_ciphers will always be in the conf, just check if value is not "" 

For example: 

if api_conf['https']['ssl_ciphers']: